### ISSUE TITLE
Adding new django tests to run after a Cantus Ultimus instance first launches

### ISSUE THREAD

**Issue by napulen** [DDMAL/cantus#398]
We can use the django tutorial for automated tests as a reference: https://docs.djangoproject.com/en/3.1/intro/tutorial05/

Things that would make for good tests:

- Make sure that making an http request to all of the static pages (e.g., `home`, `about`, `team`) returns some "proper" HTML content. (Left to decide what "proper HTML content" means in this case)
- Make sure that importing the chants of a well-known manuscript from Cantus DB results in the expected number of chants

**Comment by eliwaksbaum** [comment]
@napulen is there a problem with the existing tests? I've been working on trying to make those run

**Comment by napulen** [comment]
@eliwaksbaum I forgot about them! 

Some of them are obsolete. The ones under `mei_conversion` and `test_gamera_xml2mei` should be ignored/removed at this point. I don't think we'll reuse that MEI workflow anymore.

The one under `management` is relevant but outdated.

The `models` tests seem good, worth adapting. Same for `views`. I'd say, in order of priority:

1. `models`
2. `views`
3. `management`

**Comment by napulen** [comment]
Thanks for noticing this!

**Comment by eliwaksbaum** [comment]
```
System check identified 1 issue (0 silenced).
.F.....Deleting old concordances data...
Successfully imported 12 concordances into database.
Waiting for Solr to finish...
Done.
.....................Not Found: /chant/2f63f986449349769d7a313e0fc6edb3/
......Not Found: /concordance/2f63f986449349769d7a313e0fc6edb3/
...Not Found: /chant/2f63f986449349769d7a313e0fc6edb3/
..Not Found: /folio-set/manuscript/3/66666666666/
....Not Found: /manuscript/2f63f986449349769d7a313e0fc6edb3/
.Not Found: /manuscript/2/
...
======================================================================
FAIL: test_expand_genre (cantusdata.test.helpers.test_expandr.ExpandrFunctionsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/code/public/cantusdata/test/helpers/test_expandr.py", line 44, in test_expand_genre
    self.assertEqual(expandr.expand_genre("Pr"), "Prefatio")
AssertionError: 'Prefatio (when written out and notated)' != 'Prefatio'
- Prefatio (when written out and notated)
+ Prefatio

----------------------------------------------------------------------
Ran 47 tests in 54.166s
```
Got the existing tests to run without errors, here's the output of the existing suite. Achieved with the  latest commits from #404

**Comment by eliwaksbaum** [comment]
@napulen are we satisfied with these tests or are there others we want to add?

**Comment by napulen** [comment]
See comments on #404 

About the number of tests, we need more, but this is a great start! We can PR some more later.

**Comment by napulen** [comment]
Fixed after #406
